.classpath

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are these dot files artifacts from the editor or content for the website? If they're from the editor, I'd prefer not to keep them (since members on our team may use different development tools). It looks like the .classpath, .project, and .settings/ folders all are generated by an editor and don't contain site content.

Reply:
Done!